modules/encyclopedia.nf

```diff
@@ -81,10 +132,11 @@ process ENCYCLOPEDIA_GLOBAL {
             path(local_dia_files),
             path(local_feature_files),
             path(local_encyclopedia_files),
+            path(library_file)
         )
-        path(library_file)
         path(fasta_file)
         val output_postfix
+        val align
```

Comment:
What is this align variable for?

Reply:
When I was doing a deeper dive into EncylcopeDIA, I realized that the `-libexport` subcommand has a `-a` option to enable retention time alignment. For chromatogram library building we want this disabled, but we want it enabled for our quant runs. So, instead creating a duplicate process, I just added this internal parameter.